tor/controller.go

```diff
@@ -77,6 +78,14 @@ var (
 	// errTCNotStarted is used when we require the tor controller to be
 	// not stopped while it is.
 	errTCStopped = errors.New("tor controller must not be stopped")
+
+	// replyFieldRegexp is the regular expression used to find fields in a reply.
+	// Parameters within a reply should be of the form KEY=VALUE or KEY="VALUE",
+	// where quoted values might contain spaces, newlines and quoted pairs. If
+	// the parameter doesn't contain "=", then we can assume it doesn't provide
+	// any relevant information that isn't already known. Read more on this topic:
+	//   https://gitweb.torproject.org/torspec.git/tree/control-spec.txt#n188
+	replyFieldRegexp = regexp.MustCompile(`[^" \n]*=(?:(?:"(?:[^"\\]|\\[\0-\x7F])*")|(?:[^" \n]*))`)
```

Comment:
Looks like this can be simplified as:
```
`[^" \n]*=(?:"(?:[^"\\]|\\[\0-\x7F])*"|[^" \n]*)`
```

Also, don't we expect at least one character before the `=` character? So it should be `[^" \n]+` instead of `[^" \n]*`?

Reply:
You're right. I also added `\r` to the list, since Tor uses CRLF. Somehow missed that when I was reading the spec.